Makefile

```diff
@@ -379,7 +379,7 @@ multimod-verify: install-tools
 
 .PHONY: multimod-prerelease
 multimod-prerelease: install-tools
-	multimod prerelease -v ./versions.yaml -m collector-core
+	multimod prerelease -b=false -v ./versions.yaml -m collector-core
```

Comment:
"-s=true" to skip `go mod tidy` from the tool?

https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/main/multimod/cmd/prerelease.go#L83

Reply:
👍🏻 done